.eslintignore

```diff
@@ -1,5 +1,9 @@
 node_modules/*
+
+/* If you remove this next line, the tests will fail with
```

Comment:
What is the reason for this side-effect?  Do we understand it, and can we avoid it some other way?  I think we would like to lint the generated migrations ideally (although that could be done separately in its own test).